src/System.AppContext/src/System.AppContext.csproj

```diff
@@ -22,7 +22,8 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net463_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net463_Release|AnyCPU'" />
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)'=='true'">
-    <TargetingPackReference Include="mscorlib" />
+    <TargetingPackReference Include="mscorlib" Condition="'$(TargetGroup)' == 'net463'" />
```

Comment:
You can actually remove these TargetingPackReference's to the core assembly they should default to System.Private.CoreLib if that is present, See https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/packageresolve.targets#L148. You could also explicitly setTargetingPackCoreAssembly property if needed.